tools/utils/policy_test

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env python3
+import os
+import argparse
+import json
+import yaml
+import tests.common
```

Comment:
I'm not clear how this is working, unless its being executed only in the root of a checkout?

Reply:
Opps, good call. I've been only doing that.

Reply:
Ah, the problem here is that tests aren't installed as part of the `c7n` package. So we'd need to either extract that logic into `c7n.testing` or move all the testing code into the main package. Thoughts?

Reply:
yeah thats related to my other comment thread on this pr, I'm still a bit unclear. there's also the issue that this is effectively provider specific, we're looking at refactoring a c7n_core out of c7n to avoid the dependencies in the other providers. it might be worthwhile to pause for a moment and actually write up an issue/specification of what we want unit testing to look like bonus, it will double as documentation, but it gives a chance also for the other provider contributors to comment.

Reply:
Ack. Is there currently an issue open on this? I can open up a tix if that's what you mean.